x-pack/plugins/infra/server/routes/log_sources/configuration.ts

```diff
@@ -85,11 +85,13 @@ export const initLogSourceConfigurationRoutes = ({ framework, sources }: InfraBa
           ? sources.updateSourceConfiguration(
               requestContext.core.savedObjects.client,
               sourceId,
+              // @ts-ignore
```

Comment:
I've already added the same instructions during https://github.com/elastic/kibana/pull/113950
The problem is that an interface incompatibility error is highly unstable. `scripts/type_check` might fail locally, but pass on CI